lib/remove_bg/rate_limit_info.rb

```diff
@@ -15,16 +17,17 @@ def initialize(headers)
 
     def reset_at
       return if reset_timestamp.nil?
+
       Time.at(reset_timestamp).utc
     end
 
     def to_s
-      "<RateLimit"\
-      " reset_at='#{reset_at.iso8601}'"\
-      " retry_after_seconds=#{retry_after_seconds}"\
-      " total=#{total}"\
-      " remaining=#{remaining}"\
-      ">"
+      "<RateLimit " \
+        "reset_at='#{reset_at.iso8601}' " \
+        "retry_after_seconds=#{retry_after_seconds} " \
+        "total=#{total} " \
+        "remaining=#{remaining}" \
+        ">"
```

Comment:
I think changes the output format.

Reply:
@SxDx no, it just moves the space from the beginning of the line to the end of the line.

The rule is `Layout/LineContinuationLeadingSpace`.